docs/api/field_type_type_and_value.md

```diff
@@ -45,19 +45,8 @@ This will also apply to all user interfaces and the REST API, which therefore mu
 
 ### Field Type name
 
-To be able to generate a Content item name when the Field is part of a name schema or a URL schema,
-implement `eZ\Publish\SPI\FieldType\Nameable` and register this service using the `ezpublish.fieldType.nameable` tag.
-
-The `eZ\Publish\SPI\FieldType\Nameable::getFieldName` method will be used to retrieve the name.
-
-``` yaml
-ezpublish.fieldType.ezobjectrelation.nameable_field:
-    class: '%ezpublish.fieldType.ezobjectrelation.nameable_field.class%'
-    arguments:
-      - '@ezpublish.spi.persistence.cache.contentHandler'
-    tags:
-        - {name: ezpublish.fieldType.nameable, alias: ezobjectrelation}
-```
+To be able to retrieve a Content item name when the Field is part of a name schema or a URL schema,
+use `eZ\Publish\Core\FieldType\FieldType::getName` method.
```

Comment:
We need to hint against SPI interface here: `eZ\Publish\SPI\FieldType\FieldType::getName`. Moreover this is no longer optional, so I would rephrase this sentence, something like
```
Field name is used when the Field is part of a name schema or a URL schema to generate Content item name or URL alias respectively. It is retrieved by calling `eZ\Publish\SPI\FieldType\FieldType::getName` method which must be implemented.
```
feel free of course to fix that the way it suits you ;)